### PR TITLE
Implement showDoubleView prop

### LINKED ISSUE #82: Showing two months at a time for range selection
Thanks for this awesome component. 🎉  The no styling part makes this the best available solution for me. ❤️ 

I wanted to know if there is a way to show two months side by side for range view as generally, it makes sense if we are selecting a range.

I want something just like this.

screen shot 2018-06-12 at 11 36 09 pm

### PR DESCRIPTION
**Pull request by wojtekmaj**
Closes #82

### CODE REVIEW

File: src/Calendar.jsx [hunk 1]
```diff
@@ -441,7 +450,7 @@ export default class Calendar extends Component {
             onClick={mergeFunctions(clickAction, onClickDay)}
             onClickWeekNumber={onClickWeekNumber}
             onMouseLeave={onMouseLeave}
-            showFixedNumberOfWeeks={showFixedNumberOfWeeks}
+            showFixedNumberOfWeeks={showFixedNumberOfWeeks || showDoubleView}
```
Comment (frankleng) [inline src/Calendar.jsx:453]: @wojtekmaj  thanks for the great lib!
why was `showFixedNumberOfWeeks` forced to `true` in double view?
mind if I PR to remove this?

Comment (wojtekmaj) [reply src/Calendar.jsx:453]: This is because otherwise the UI would look quite bad with different amount of rows on each side.

Comment (frankleng) [reply src/Calendar.jsx:453]: @wojtekmaj  got it. so it's subjective. ex. priceline.com implements a dual view without locking down # of weeks. 
are you open to a PR to remove this?

Comment (wojtekmaj) [reply src/Calendar.jsx:453]: I like this idea.

**Pull request merged**